src/screens/AppIntroScreen.tsx

```diff
@@ -106,6 +107,6 @@ const styles = StyleSheet.create({
   sliderButton: {
     borderBottomColor: colors.darkText,
     borderBottomWidth: 1,
-    marginTop: 14 // no normalization here as the dots position does not use normalization either
+    marginVertical: 16 // no normalization here as the dots position does not use normalization either
```

Comment:
12 makes it more aligned with the dots in my test than 16?

Reply:
hmm. when testing it made it look better with 16 and both top and bottom, instead of only top.

that is also the margin that is used for the dots component internally. 🤔

Reply:
i should have attached screenshots of my test yesterday. will contact you on this one later

Reply:
|16|14|12|
|---|---|---|
|![IMG_0019](https://user-images.githubusercontent.com/1942953/144294240-83b14cb3-70b9-4fb6-83d4-8da3cdb37f1f.PNG)|![IMG_0020](https://user-images.githubusercontent.com/1942953/144294247-ffb88cca-644a-4e76-aae4-0add3a75b374.PNG)|![IMG_0021](https://user-images.githubusercontent.com/1942953/144294248-2ce41b6d-4231-47af-b7e6-4e05f8b393f4.PNG)|

Reply:
screenshots taken with iPhone 6s real device.
comparing them (full screen after clicking the image), i think the 12 aligns best? 

<img width="1531" alt="Bildschirmfoto 2021-12-01 um 19 42 55" src="https://user-images.githubusercontent.com/1942953/144294442-96d04880-26f4-44bd-9024-7e602cb156d8.png">

Reply:
✅